CWE/events/italy.txt

```diff
@@ -455,6 +455,1043 @@ country_event = {
+	}
+
+	mean_time_to_happen = { months = 2 }
+
+	option = {
+		name = "Democratic Centre (DC-PSDI-PLI-PRI)"
+		ruling_party_ideology = conservative
+		any_pop = { 
+			ideology = { factor = 0.1 value = conservative }
+			ideology = { factor = 0.05 value = big_tent }
+			ideology = { factor = 0.02 value = liberal }
+		}
+		
+		political_reform = jefferson_method
+		
+		set_country_flag = 1953_ITA_Legge_Truffa
+		ai_chance = { factor = 0.2 }
+	}
+	
+	option = {
+		name = "One Party DC (DC)"
+		ruling_party_ideology = conservative
+		any_pop = { 
+			ideology = { factor = -0.1 value = conservative }
+			ideology = { factor = 0.02 value = nationalist }
+		}
+		any_pop = { 
+			limit = { strata = poor }
+			ideology = { factor = 0.03 value = communist }
+		}
+		any_pop = { 
+			limit = { OR = { strata = poor strata = middle } }
+			ideology = { factor = 0.03 value = socialist }
+		}
+		set_country_flag = 1953_ITA_Centrism
+		ai_chance = { factor = 0.7 }
+	}
+
+	option = {
+		name = "Social-communist Front (PCI-PSI)"
+		ruling_party_ideology = communist
+		any_pop = { 
+			limit = { strata = poor }
+			ideology = { factor = 0.2 value = communist }
+		}
+		any_pop = { 
+			limit = { OR = { strata = poor strata = middle } }
+			ideology = { factor = 0.1 value = socialist }
+		}
+		set_country_flag = 1953_ITA_FDP
+		ai_chance = { factor = 0.1 }
+	}
+
+}
+
+#The Fall of De Gasperi VIII
+country_event = {
+  	id = 8020313
+  	title = "The Fall of De Gasperi VIII"
+  	desc = EVT_8020313_DESC
+  	picture = "de_gasperi"
+  
+  	fire_only_once = yes
+
+	trigger = {
+		tag = ITA
+		year = 1953  
+		NOT = { year = 1958 }
+		
+		government = democracy
+		
+		ruling_party_ideology = conservative
+		
+		has_country_flag = parliamentary_system_republic
+		
+		has_country_flag = 1953_ITA_Centrism
+	}
+
+	mean_time_to_happen = { months = 1 }
+
+	option = {
+		name = "Pella (DC PLI-PNM)"
+		
+		#Leadership
+		remove_country_modifier = expert_diplomat
+		remove_country_modifier = successful_industrialist
+		remove_country_modifier = ivory_tower_intellectual
+		remove_country_modifier = man_of_the_people
+		remove_country_modifier = landed_gentleman
+		remove_country_modifier = military_man
+		remove_country_modifier = lord_admiral
+		remove_country_modifier = raving_loon
+		remove_country_modifier = luddite
+		remove_country_modifier = great_statesman
+		remove_country_modifier = backroom_boy
+		remove_country_modifier = great_reformer
+		remove_country_modifier = gauche_buffoon
+		remove_country_modifier = administrative_genius
+		remove_country_modifier = visionary_thinker
+		remove_country_modifier = competent_placeholder
+		remove_country_modifier = imperious_autocrat
+		remove_country_modifier = prince_of_terror
+		remove_country_modifier = efficient_sociopath
+		remove_country_modifier = pig_headed_isolationist
+		remove_country_modifier = ambitious_union_boss
+		remove_country_modifier = dour_economist
+		
+		add_country_modifier = administrative_genius
```

Comment:
This is wrong. It should be add_country_modifier = { name = administrative_genius duration = -1 }